src/stats.c

```diff
@@ -52,6 +52,23 @@ char opt_stats_print_opts[stats_print_tot_num_options+1] = "";
 
 /******************************************************************************/
 
+static uint64_t
+rate_per_second(uint64_t value, uint64_t uptime_ns) {
+	static uint64_t billion = 1000000000;
+	if (uptime_ns == 0 || value == 0) {
+		return 0;
+	}
+	if (uptime_ns < billion) {
+		if (opt_abort) {
+			assert(value * billion > value);
+		}
+		return value * billion / uptime_ns;
```

Comment:
Well I changed my mind here; let's simply consider the subsecond case as 1 second, i.e. 
  if (uptime_ns < billion) {
       return value; 
  }